source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -53,11 +117,37 @@ A Service Account is created in the [namespace creation github repository](https
   token: ZX...EE=
 ```
 
+There will be a secret for the circle ci service account. The secret contains the ca.crt and token which we will be injected into the circle CI image container as environment variables. This will allow it to authenticate against the cluster. 
+Access to the cluster is scoped for the namespace the service account was created in. 
+
+Once the circle CI service account is created above, you can grab the ca.crt as follows:
+
+```bash
+k get secret <NAME-OF-CIRCLECI-SERVICE-ACCOUNT-SECRET> -o yaml -n <NAMESPACE | grep ca.crt
+```
+
+Likewise grab the circle CI service accont token as follows: 
+
+```bash
+k get secret <NAME-OF-CIRCLECI-SERVICE-ACCOUNT-SECRET> -o yaml -n <NAMESPACE | grep token
+
+The token will need to be decoded as follows: 
+
+echo <ENCODED_TOKEN> | base 64 --decode
+```
+
+Both the ca.crt and token will be added as environment variables by creating a context in the circle CI console, that holds these values as shown below. 
```

Comment:
"will be added" makes it sound as if some other process is going to take care of this for us. I'd suggest something like; "We need to add both the ..."